consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/api/IClaimDefencesService.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.moj.sdt.cmc.consumers.api;
+
+import uk.gov.moj.sdt.cmc.consumers.model.SubmitQueryResponse;
+
+public interface IClaimDefencesService {
+
+    SubmitQueryResponse claimDefences(String authorization, String serviceAuthorization, String idamId,
```

Comment:
instead of SubmitQueryResponse from here, can we do return ClaimDefencesResponse from this method.
SubmitQueryResponse is the response of SubmitQueryResponse submitQuery(...) method in CMCConsumerGateway